presto-bytecode/src/main/java/com/facebook/presto/bytecode/ClassDefinition.java

```diff
@@ -121,10 +122,16 @@ public String getSource()
         return ImmutableList.copyOf(methods);
     }
 
+    public boolean isInterface()
+    {
+        return access.contains(INTERFACE);
+    }
+
     public void visit(ClassVisitor visitor)
     {
         // Generic signature if super class or any interface is generic
         String signature = null;
+        boolean isInterface = access.contains(INTERFACE);
```

Comment:
I'm not sure calling this variable `interface` is a good idea given interface is not an adjective. Calling it `interface` is quite confusing.